test/integration/cli-volumes.test.js

```diff
@@ -37,6 +41,8 @@ test('triton volume create ...', testOpts, function (tt) {
     tt.test('  cleanup leftover resources', function (t) {
         h.triton(['volume', 'delete', '-y', '-w', validVolumeName].join(' '),
             function onDelVolume(delVolErr, stdout, stderr) {
+                // If there was nothing to delete, this will fail so that's the
+                // normal case. Too bad we don't have a --force option.
```

Comment:
I think a --force would work. At least if it worked in the `rm -f <file>` sense where we mean it shouldn't be an error if the file already doesn't exist. In this case we get an error when we're trying to delete a VM that's already not there.